client/ayon_blender/api/plugin.py

```diff
@@ -53,18 +54,21 @@ def get_unique_number(
     folder_name: str, product_name: str
 ) -> str:
     """Return a unique number based on the folder name."""
-    avalon_container = bpy.data.collections.get(AVALON_CONTAINERS)
-    if not avalon_container:
+    ayon_container = bpy.data.collections.get(AYON_CONTAINERS) or (
+        bpy.data.collections.get("AVALON_CONTAINERS")
```

Comment:
Why do we suddenly hardcode this value here instead of still using the `AVALON_CONTAINERS` constant? Seems like we're still better of keeping that constant in use?